newdle/client/src/components/ParticipantTable.js

```diff
@@ -77,17 +76,10 @@ function AvailabilityRow({availability: {startDt, available}, setActiveDate, act
       <Table.Cell width={10} className={styles['available-participants']} textAlign="left">
         <div className={styles['wrapper']}>
           <div className={styles['availability-indicator']}>
-            <CircularProgressbar
-              value={available.length}
-              text={`${available.length} / ${numberOfParticipants}`}
-              maxValue={numberOfParticipants}
-              styles={buildStyles({
-                textSize: '18px',
-                strokeLinecap: 'butt',
-                pathColor: numberOfParticipants === 0 ? 'gray' : 'lightgreen',
-                trailColor: 'red',
-                textColor: 'black',
-              })}
+            <ProgressRing
+              available={available.length}
+              ifNeeded={0}
```

Comment:
Would you mind adding another commit to use actual data here?

I think the best option for that would be changing the `getParticipantAvailability` selector to also include the number of `ifneedbe` answers.

Reply:
I made the change (and all the others) but I am not sure how to best test this one, since I was having trouble with logins (and can't accept the invite as other people).

Reply:
With `SKIP_LOGINS` you only have one account, but if you access the newdle URL without `/summary` in an incognito tab, you can submit anonymously (ie creating a new participant whenever you submit).

You can also change the email backend in the config from `...smtp...` to `...console...` and then copy the personal link and use that one to answer as an invited participant.